fusor-ember-cli/app/mirage/factories/discovered_host.js

```diff
@@ -16,8 +16,8 @@ export default Mirage.Factory.extend({
   mac: function(i) {                  // and functions
     return i + '2:54:00:0f:65:68';
   },
-  cpus: 1,
-  memory_human_size: "1.96 GB",
+  cpus: 4,
+  memory_human_size: "7.8 GB",
```

Comment:
This is just for mirage so that the resources available are higher than what is selected, so the bars are not red.